modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -585,7 +593,7 @@ public interface Preferences extends ObservableMap<String, Object> {
         /**
          * The color used for background regions.
          * <p>
-         * If the platform does not report a background color, this property defaults to {@code Color.WHITE}.
+         * If the platform does not report a background color, this property defaults to {@link Color#WHITE}.
```

Comment:
Minor: I don't think you need a link here because it's linked in the default value tag.

Reply:
Agreed. Only one one of them needs the link (and yes, it's minor).